src/main/java/uk/gov/justice/hmpps/prison/repository/sql/CaseNoteRepositorySql.kt

```diff
@@ -35,6 +35,21 @@ enum class CaseNoteRepositorySql(val sql: String) {
     """
   ),
 
+  GROUP_BY_TYPES_FOR_BOOKING_FROM_DATE(
+    """
+        SELECT CASE_NOTE_TYPE,
+        CASE_NOTE_SUB_TYPE,
+        OFFENDER_BOOK_ID  BOOKING_ID,
+        COUNT(*)          NUM_CASE_NOTES,
+        MAX(CONTACT_TIME) LATEST_CASE_NOTE
+        FROM OFFENDER_CASE_NOTES OCS
+        WHERE AUDIT_TIMESTAMP >= :fromDate
```

Comment:
Are you sure AUDIT_TIMESTAMP is the correct date for this query? Not CREATE_DATETIME?

Reply:
Yes its the one with the index on it